phpstan.neon

```diff
@@ -393,3 +393,12 @@ parameters:
         - '#Parameter \#1 \$type of method Rector\\NodeCollector\\NodeCollector\\ParsedNodeCollector<TNodeType of PhpParser\\Node\>\:\:getNodesByType\(\) expects class\-string<TNodeType of PhpParser\\Node\>, string given#'
 
         - '#Class with base "(.*?)" name is already used in "_HumbugBox(.*?)"#'
+
+        #tmp
+        -
+            message: '#Class (.*?) has invalid namespace category (.*?)\. Pick one of\: (.*?)#'
+            path: 'rules/generic/src/Rector/Architecture'
```

Comment:
Should I move rules in Architecture folder to the `rules/generic/src/Rector/` and use an appropriate namespace for it or I have to create new namespaces in Architecture folder?